test.properties

```diff
@@ -2864,7 +2864,8 @@
     playwright.projects.includes[playwright-js-tomcat90-*][commerce]=${commerce.playwright.projects}
 
     test.batch.class.names.includes[commerce]=\
-        **/commerce/**/*Test.java
+        **/commerce/**/*Test.java,\
+        **/portal-security-membership-policy-test/**/*Test.java
```

Comment:
@manuelecastro can you please add this to `test.batch.class.names.includes[um]=\` instead, as is more user management related. Thx